src/templates/Header.html

```diff
@@ -32,26 +32,14 @@ <h2 class="cta" ref="cta">Find events near you</h2>
         </button>
       </div>
 
-      <div v-if="isFilterEventsOpen">
-        <div class="filter-events" :style="filterEventsTop">
-          <div class="filter-events-header">
-            <span>Filter events</span>
-            <span class="filter-events-close" @click="toggleFilterEvents">
-              ✖
-            </span>
-          </div>
-          <div class="filter-events-body">
-            <event-date-filters :show-title="true" :filters="filters" ></event-date-filters>
-            <event-type-filters :show-title="true" :filters="filters"></event-type-filters>
-            <button
-              class="filter-events-search"
-              @click="toggleFilterEvents"
-            >
-              Search events
-            </button>
-          </div>
-        </div>
-      </div>
+      <mobile-event-filters
```

Comment:
Nothing new added here, just a straight up extraction. This way when the HTML template is initially parsed, the user doesn't see any extra junk before vue boots.